man/nvim.1

```diff
@@ -336,8 +336,22 @@ Print version information and exit.
 .El
 .Sh ENVIRONMENT
 .Bl -tag -width Fl
+.It Ev XDG_CONFIG_HOME
+Path to use for the user-local runtime directory, see
+.Sx FILES .
+Defaults to
+.Pa ~/.config
```

Comment:
*Default* for `$XDG_CONFIG_HOME` is hardcoded in the standard (here: in `stdpaths.c`) and may not change.